command/state_meta.go

```diff
@@ -23,8 +25,9 @@ func (c *StateMeta) State(m *Meta) (state.State, error) {
 		return nil, err
 	}
 
+	env := c.Env()
```

Comment:
Ah, ok. This was the reason for your question above, in that StateMeta was calling the embedded Meta.Env method, and embedding Meta in StateMeta was just the cleanest way to ensure all State commands also had the same method. 

Looking at this though, I think we can also just drop the Meta param, since `StateMeta` would now have the correctly configured Meta for the command embedded.